src/ArduinoIoTCloudTCP.cpp

```diff
@@ -661,18 +661,18 @@ ArduinoIoTCloudTCP::State ArduinoIoTCloudTCP::handle_Connected()
         /* Clear the request flag. */
         _ota_req = false;
         /* Transmit the cleared request flags to the cloud. */
-        sendClearedOTARequestToCloud();
+        sendDevicePropertyToCloud("OTA_REQ");
```

Comment:
I love this commit. Reducing code size while increasing readability. Top work! :bow:

Reply:
Thanks :muscle: